lib/server.go

```diff
@@ -190,6 +190,34 @@ func (srv *Server) GetNetworkManager() *NetworkManager {
 	return srv.networkManager
 }
 
+func (srv *Server) AdminOverrideViewNumber(view uint64) error {
```

Comment:
This clearly worked right? If so then I'd keep it and just gate it behind a JWT check, which I think you probably did in backend (reviewing that next).